src/niscope/system_tests/test_system_niscope.py

```diff
@@ -141,6 +141,24 @@ def test_fetch_measurement_stats(self, multi_instrument_session):
             assert measurement_stats[i].channel == expected_channels[i]
             assert measurement_stats[i].record == expected_records[i]
 
+    def test_fetch_array_measurement_defaults(self, multi_instrument_session):
```

Comment:
These tests pass on my machine against a locally built grpc-device with the fixes in this [grpc-device PR](https://github.com/ni/grpc-device/pull/792).

After that goes in, do I need to update nimi-bot to pull in the new grpc-device server or does that automatically get bumped @DavidCurtiss?

Reply:
You have to manually update it.
1. Run [this pipeline](https://ni.visualstudio.com/DevCentral/_build?definitionId=10158) to (indirectly) trigger an installer build, or wait for it to run automatically overnight.
2. After a new installer is subsequently [built](https://ni.visualstudio.com/DevCentral/_build?definitionId=10129), update nimibot like [this](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/392031?_a=files). (You probably don't have to worry about the feedUrl, but I always update it to latest when I update the other.)
3. Once that is in, run [this pipeline](https://ni.visualstudio.com/DevCentral/_build?definitionId=7984) to rebuild nimibot. It's broken right now, though. I posted about that [here](https://teams.microsoft.com/l/message/19:0fe83ae07a1341599eee52a965b9655d@thread.tacv2/1668545120515?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=512984dc-3cc8-4e79-b103-9cf4cf2b5821&parentMessageId=1668545120515&teamName=RDSS%20Compute%20Farm%20Infrastructure&channelName=Support&createdTime=1668545120515&allowXTenantAccess=false).

Reply:
I have the pipeline for step 3 running now. When that finishes I'll re-queue this PR build to run against the new nimibot.